test/SlowTests/Issues/RavenDB-15900.cs

```diff
@@ -56,8 +56,7 @@ public async Task RemoveEntryFromRaftLogEP()
 
             // Stuck leader on this command
             var testCmd = new RachisConsensusTestBase.TestCommandWithRaftId("test", RaftIdGenerator.NewId());
-            await Assert.ThrowsAsync<UnknownClusterCommandException>(() => leader.ServerStore.SendToLeaderAsync(testCmd));
-
+            _ = leader.ServerStore.SendToLeaderAsync(testCmd);
```

Comment:
this doesn't throw any more?
Also, this is now racy with the next line, no?

Reply:
I fixed the throwing problem, will fix in the next commit..
The 'TestCommandWithRaftId' command is never being committed, so it isn't racy.